foyer/tests/test_forcefield.py

```diff
@@ -105,14 +149,6 @@ def test_write_refs_multiple():
                                              n=0))
     assert not diff
 
-@pytest.mark.skipif(not has_mbuild, reason="mbuild is not installed")
-def test_write_bad_ref():
```

Comment:
Any reason to have dropped this test?

Reply:
We are already faking the original response, so, it made sense since a point of failure would be the `request`. We could use same tricks to generate user warnings. I have no opinion one way or another.

Reply:
I misunderstood this test at first, I think it's fine to remove it.